src/puzzlepieces/settings.py

```diff
@@ -88,7 +89,6 @@
         "PASSWORD": os.environ.get("SQL_PASSWORD", "puzzling"),
         "HOST": os.environ.get("SQL_HOST", "localhost"),
         "PORT": os.environ.get("SQL_PORT", "5432"),
-        "CONN_MAX_AGE": 60,
```

Comment:
could you elaborate? is the default setting better?

Reply:
We had to remove this from our running instance. We were getting server error 500 intermittently with it, when under load. Default, we started working again.